perfkitbenchmarker/packages/fio.py

```diff
@@ -47,31 +56,6 @@ def AptInstall(vm):
   _Install(vm)
 
 
-def ExtractFioParameters(fio_parameter):
-  """Extract fio parameters from raw string.
-
-  Sample parameter_string:
-  overwrite=0
-  rw=write
-  blocksize=512k
-  size=10*10*1000*$mb_memory
-  iodepth=64
-  direct=1
-  end_fsync=1
-
-  Args:
-    fio_parameter: string. Parameters in string format.
-  Returns:
-    A dictionary of parameters.
-  """
-  parameters = regex_util.ExtractAllMatches(
-      PARAMETER_REGEX, fio_parameter)
-  param_dict = {}
-  for parameter in parameters:
-    param_dict[parameter[0]] = parameter[1]
-  return param_dict
-
-
 def ParseJobFile(job_file):
```

Comment:
There is already a test called testParseFioJobFile under tests/packages/fio_test.py